notebooks/filtering.ipynb

```diff
@@ -5,40 +5,567 @@
    "id": "78cee78b",
```

Comment:
<p>This cell crashed when I tried to run it initially. I was using the smallest RAM size for the server, and when I increased to 7.4 GB it worked. We should specify what resources are needed somewhere in the setup information.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ICESAT-2HackWeek/icesat2-cookbook/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>